CHANGELOG.md

```diff
@@ -29,6 +29,11 @@ Main (unreleased)
   This port can be configured with `--server.http.listen-addr` or using
   the default listen address`127.0.0.1:12345`. (@mattdurham) 
 
+- Fix clustering on instances running within Istio mesh by allowing to change the name of the clustering port
```

Comment:
We don't put change entries for the Helm chart here since this tracks changes for Alloy releases. Can you move this entry to the [Helm-specific CHANGELOG](https://github.com/grafana/alloy/blob/main/operations/helm/charts/alloy/CHANGELOG.md) instead?

Reply:
Moved